src/plugins/visible-range/index.js

```diff
@@ -118,7 +118,7 @@ export default class VisibleRangePlugin extends AbstractPlugin {
       // eslint-disable-next-line no-param-reassign
       range = null;
     }
-    // longitude range is between 0 and 2*PI
+    // longitude range is between 0 (center of panorama) and 2*PI
```

Comment:
```suggestion
    // longitude range is between 0 and 2*PI
```

this is true for the whole library, not only here

Reply:
Right. Would you be ok with adding a comment about it in docs/guide/config.md (if that's the right place to put it)?